### DISCUSSION TITLE
Support a file list as SRC

### DISCUSSION THREAD

**Discussion by louie007** [juicedata/juicesync#123] [General]
Hi @davies, thanks for sharing this great tool juicesync !
I'm wondering if you can add a new feature like supporting a plain text file as SRC ?

eg.
s3://bucket1/file1
oss://buckert2/Dir1/
etc

**Comment by davies** [comment]
Right now, you can use xargs

```
cat mylist.txt | xargs -I{}  juicesync {} DST
```

**Reply by louie007** [reply]
Thx bro, it works
